doc/classes/Light2D.xml

```diff
@@ -340,46 +340,67 @@
 	</methods>
 	<members>
 		<member name="color" type="Color" setter="set_color" getter="get_color">
+			The Light2D's [Color].
 		</member>
 		<member name="editor_only" type="bool" setter="set_editor_only" getter="is_editor_only">
+			If [code]true[/code] Light2D will only appear when editing the scene. Default value: [code]false[/code].
```

Comment:
(Opinion) I kinda prefer a comma after 'If true'.

Reply:
Most English style guides say not do use it after a brief introductory phrase (< 4 words).

Reply:
Noted. I just want it to be consistent throughout the docs.

Reply:
Well, we can put no comma in the style guide, but that hasn't been merged and published yet.